masonry_core/src/doc/06_masonry_concepts.md

```diff
@@ -51,6 +52,15 @@ Masonry should force the widget to lose pointer capture when some events occur;
 Examples of use cases for pointer capture include selecting text, dragging a slider, or long-pressing a button.
 
 
+## Active
+
+An "active" widget is one that the user is currently interacting with.
+
+Currently, a widget is determined to be active when it has pointer capture, though that definition could change.
```

Comment:
Again ... is there some anticipation that this might change? Anything might change, so why does this need to call this out specifically?